### PR TITLE
Add css class for old articles: sis-legacy

### PR DESCRIPTION
**Pull request by alexk1204**
Add css class for old articles: sis-legacy

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,7 +11,6 @@ node_modules
 *.log
 /dist/*
 /_sqldata/*
-temp
```
Comment (khawkins98) [inline .gitignore:14]: Could we keep ignoring the /temp directory, or is there a reason you want to track it in git?

File: wp-content/themes/vf-wp-sis/partials/head.php [hunk 2]
```diff
@@ -39,5 +39,10 @@
     }
 }
 
+$articleMigrated = get_field('art_migrated_from_drupal');
+if($articleMigrated === true){
+    $cssClass .= ' sis-article-legacy';
```
Comment (khawkins98) [inline wp-content/themes/vf-wp-sis/partials/head.php:44]: I've also changed this to `sis-article-legacy`, I assume you don't mind?

Comment (alexk1204) [reply wp-content/themes/vf-wp-sis/partials/head.php:44]: Yes, no problem :-)

File: .gitignore (hunk 1 above)
Comment (alexk1204) [reply .gitignore:14]: I think this was a mistake. Sorry.

File: wp-content/themes/vf-wp-sis/partials/head.php (hunk 2 above)
Comment (alexk1204) [reply wp-content/themes/vf-wp-sis/partials/head.php:44]: Please merge it if you are ready. I could deploy it later on dev.beta if you want.

**Pull request merged**